R/design_print_summary.R

```diff
@@ -0,0 +1,264 @@
+
```

Comment:
I usually try to keep reorganization to a seperate PR, otherwise the diffs get large and hard to follow, and history breaks.

Reply:
ah, thanks, tried to avoid and kept in a separate PR. will do moving forward.